pylab/settings/base.py

```diff
@@ -175,6 +175,7 @@
 )
 
 INSTALLED_APPS += (
+    'pylab.core',
```

Comment:
> When models are defined in single place it is easier to manage dependencies between
different code parts.
Each app should have description, what should and what should not be included in it. Since our website will have some "sub-websites", clear functionality logic distribution would really help. It could be brief description in pylab/README.rst or each app could have its own README.rst file. E.g. can ```website``` app include any kind of model, or all of them should be in ```core``` app?